style.css

```diff
@@ -335,3 +333,196 @@ textarea {
   background-color: #fff;
   margin: 0 0 100px 24px;
 }
+
+@media (min-width: 992px) {
+  body {
+    max-width: 992px;
+  }
+
+  .burger {
+    display: none !important;
+  }
+
+  .desktop-only {
+    display: flex;
+    margin: 25px 32px 32px 0;
+    color: #344563;
+    font-family: Poppins, sans-serif;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  nav {
+    margin-left: 115px;
+  }
+
+  .headline-content {
+    background-image: url("./Header\ bg\ \(1\).svg");
+    background-repeat: no-repeat;
+    background-size: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    justify-items: center;
+    padding-left: 150px;
+    border-bottom-left-radius: 20px;
+  }
+
+  .headline-content > h1 {
+    width: fit-content;
+    height: 122px;
+    margin: 0;
+    font-size: 35px;
+    font-weight: bold;
+    color: #091e42;
+  }
+
+  li p {
+    width: fit-content;
+  }
+
+  .headline-content > p {
+    width: 650px;
+    height: 98px;
+    font-size: 15px;
+    line-height: 20px;
+    letter-spacing: 0.3px;
+  }
+
+  h3 {
+    width: 920px;
+    height: 24px;
+  }
+
+  .socials {
+    width: 800px;
+    height: 20px;
+    padding-bottom: 20px;
+  }
+
+  .works {
+    display: grid;
+    max-width: 992px;
```

Comment:
- [ ] Setting a max-width of your works element to 992px will cause your section to cut halfway on the visitor's device.
I will suggest you remove the max-width to get a full 100% width.